doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1551 @@
+  * Migration of the Device Attestation Certificates private key to Key Management Unit (KMU) for the nRF54L Series SoCs.
+    See :ref:`matter_platforms_security_dac_priv_key_kmu` to learn how to enable it in your sample.
+  * Support for the Device Firmware Upgrade over Bluetooth LE SMP and Matter Over-the-air Software Update (OTA) on nRF54H20 SoC.
+
+* Updated:
+
+  * The default Trusted Storage AEAD key to Hardware Unique Key (HUK) for supported nRF54L Series devices.
+  * Renamed the ``CONFIG_CHIP_FACTORY_RESET_ERASE_NVS`` Kconfig option to :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_SETTINGS`.
+    The new Kconfig option now works for both NVS and ZMS file system backends.
+  * The firmware version format used for informational purposes when using the :file:`VERSION` file.
+    The format now includes the optional ``EXTRAVERSION`` component.
+  * Storing the Device Attestation Certificates private key in the Trusted Storage library to be enabled for all platforms that support the PSA crypto API.
+    See :ref:`matter_platforms_security_dac_priv_key_its` for more information.
+  * The stack sizes for OpenThread and CHIP (Matter) threads to increase them for the targets using the CRACEN crypto backend.
+  * The IPv6 multicast subscription mechanism for Wi-Fi to enable the MLDv2 join procedure and improve reliability in multicast communication.
+  * Storing the Device Attestation Certificates private key in the Key Management Unit (KMU) to be enabled for nRF54L Series devices.
+    See :ref:`matter_platforms_security_dac_priv_key_kmu` for more information.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.3.0.0`` tag.
+
+Thread
+------
+
+* Added the :ref:`ug_thread_build_report` and described how to use it.
+* Updated the default Trusted Storage AEAD key to Hardware Unique Key (HUK) for supported nRF54L Series devices.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.5.0 and platform v5.1.6 (``v3.11.5.0+5.1.6``).
+    They contain a fix for the ZBOSS traces.
+    For details, see :ref:`zboss_changelog`.
+  * :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.4.
+
+* Fixed the :file:`zb_add_ota_header.py` script not being able to handle an ``APPLICATION_VERSION_STRING`` which includes a tweak, such as ``1.0.0+3``.
+
+Wi-Fi
+-----
+
+* Added:
+
+   * The :ref:`ug_nrf70_developing_offloaded_raw_tx` feature.
+   * Support for :ref:`WPA2-EAP-TLS <ug_nrf70_wifi_advanced_security_modes>` enterprise security mode.
+   * Support for :ref:`Platform Security Architecture (PSA) crypto APIs <ug_nrf70_developing_wifi_psa_support>` for WPA2™ security profiles.
+
+* Updated:
+
+   * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
+   * The WPA supplicant now uses ``kernel heap`` instead of ``application (libc) heap``.
```

Comment:
```suggestion
* Updated:

  * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
  * The WPA supplicant now uses ``kernel heap`` instead of ``application (libc) heap``.
```